samoa-api/src/main/java/org/apache/samoa/moa/core/Measurement.java

```diff
@@ -36,14 +36,19 @@
   private static final long serialVersionUID = 1L;
 
   protected String name;
-
   protected double value;
+  protected int fractionDigits;
 
   public Measurement(String name, double value) {
```

Comment:
changed to have the two parameter constructor above call the three parameter one defaulting third parameter fractionDigits to 3